lib/smart_answer_flows/energy-grants-calculator.rb

```diff
@@ -470,6 +470,9 @@ def define
         precalculate :flat_type do
           flat_type
         end
+        precalculate :under_green_deal do
```

Comment:
I this `under_green_deal` will return a boolean.. is there a case against not using a question mark at the end of this definition `under_green_deal?` For me this reads a lot more nicer.